cli/src/cmd/forge/script/sequence.rs

```diff
@@ -110,6 +169,21 @@ impl ScriptSequence {
                 )?),
                 &self,
             )?;
+
+            // cache folder writes
+            //../run-latest.json
+            serde_json::to_writer_pretty(
+                BufWriter::new(fs::create_file(&self.sensitive_path)?),
+                &script_sequence_sensitive,
+            )?;
+            //../run-[timestamp].json
+            serde_json::to_writer_pretty(
+                BufWriter::new(fs::create_file(
+                    sensitive_path.replace("latest.json", &format!("{}.json", self.timestamp)),
+                )?),
+                &script_sequence_sensitive,
+            )?;
+
             shell::println(format!("\nTransactions saved to: {path}\n"))?;
```

Comment:
maybe we should print an extra message indicating sensitive info has been written to a separate path?